core/commonMain/src/kotlinx/serialization/descriptors/SerialDescriptors.kt

```diff
@@ -24,22 +25,24 @@ import kotlin.reflect.*
  *     val nullableInt: Int?
  * )
  * // Descriptor for such class:
- * SerialDescriptor("my.package.Data") {
+ * buildClassSerialDescriptor("my.package.Data") {
  *     // intField is deliberately ignored by serializer -- not present in the descriptor as well
  *     element<Long>("_longField") // longField is named as _longField
- *     element("stringField", listSerialDescriptor<String>())
+ *     element("stringField", listSerialDescriptor<String>()) // or ListSerializer(String.serializer()).descriptor
```

Comment:
By the way, are those intrinsified?

Reply:
Given they use `serializer<T>()`, this is likely to happen